[2023-12-28T16:09:06.957+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:09:06.959+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:09:07.225+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:09:07.228+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:09:07.407+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-18T04:00:00+00:00 [queued]>
[2023-12-28T16:09:07.473+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-18T04:00:00+00:00 [queued]>
[2023-12-28T16:09:07.475+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-28T16:09:07.681+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-18 04:00:00+00:00
[2023-12-28T16:09:07.699+0000] {standard_task_runner.py:57} INFO - Started process 562 to run task
[2023-12-28T16:09:07.718+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-18T04:00:00+00:00', '--job-id', '767', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmpxichqz_9']
[2023-12-28T16:09:07.729+0000] {standard_task_runner.py:85} INFO - Job 767: Subtask Create_Table
[2023-12-28T16:09:07.938+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:09:07.940+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:09:07.945+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-18T04:00:00+00:00 [running]> on host d67dd4682ccf
[2023-12-28T16:09:08.267+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-18T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-18T04:00:00+00:00'
[2023-12-28T16:09:08.307+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-28T16:09:12.824+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/airQuality.py", line 133, in createTableSQL
    cursor.execute(createQuery)
psycopg2.errors.UniqueViolation: duplicate key value violates unique constraint "pg_class_relname_nsp_index"
DETAIL:  Key (relname, relnamespace)=(air_quality_id_seq, 2200) already exists.

[2023-12-28T16:09:12.854+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=air_quality, task_id=Create_Table, execution_date=20231218T040000, start_date=20231228T160907, end_date=20231228T160912
[2023-12-28T16:09:12.887+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 767 for task Create_Table (duplicate key value violates unique constraint "pg_class_relname_nsp_index"
DETAIL:  Key (relname, relnamespace)=(air_quality_id_seq, 2200) already exists.
; 562)
[2023-12-28T16:09:12.933+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-12-28T16:09:13.000+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-12-28T16:52:11.255+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:52:11.260+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:52:11.428+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:52:11.431+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:52:11.684+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-18T04:00:00+00:00 [queued]>
[2023-12-28T16:52:11.756+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-18T04:00:00+00:00 [queued]>
[2023-12-28T16:52:11.757+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-28T16:52:11.849+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-18 04:00:00+00:00
[2023-12-28T16:52:11.864+0000] {standard_task_runner.py:57} INFO - Started process 356 to run task
[2023-12-28T16:52:11.890+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-18T04:00:00+00:00', '--job-id', '916', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmpvwyzgxme']
[2023-12-28T16:52:11.913+0000] {standard_task_runner.py:85} INFO - Job 916: Subtask Create_Table
[2023-12-28T16:52:12.204+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:52:12.206+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:52:12.225+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-18T04:00:00+00:00 [running]> on host d67dd4682ccf
[2023-12-28T16:52:12.931+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-18T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-18T04:00:00+00:00'
[2023-12-28T16:52:12.983+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-28T16:52:13.300+0000] {airQuality.py:139} INFO - Table air_quality Already Exist
[2023-12-28T16:52:13.425+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-28T16:52:13.517+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231218T040000, start_date=20231228T165211, end_date=20231228T165213
[2023-12-28T16:52:13.634+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-28T16:52:13.766+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-12-28T16:55:09.231+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:55:09.234+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:55:09.374+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:55:09.376+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:55:09.530+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-18T04:00:00+00:00 [queued]>
[2023-12-28T16:55:09.604+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-18T04:00:00+00:00 [queued]>
[2023-12-28T16:55:09.606+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-28T16:55:09.669+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-18 04:00:00+00:00
[2023-12-28T16:55:09.688+0000] {standard_task_runner.py:57} INFO - Started process 1305 to run task
[2023-12-28T16:55:09.713+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-18T04:00:00+00:00', '--job-id', '1149', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmprrec9t6h']
[2023-12-28T16:55:09.718+0000] {standard_task_runner.py:85} INFO - Job 1149: Subtask Create_Table
[2023-12-28T16:55:09.956+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:55:09.959+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-18T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:55:09.973+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-18T04:00:00+00:00 [running]> on host d67dd4682ccf
[2023-12-28T16:55:10.382+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-18T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-18T04:00:00+00:00'
[2023-12-28T16:55:10.456+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-28T16:55:12.311+0000] {airQuality.py:139} INFO - Table air_quality Already Exist
[2023-12-28T16:55:12.439+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-28T16:55:12.543+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231218T040000, start_date=20231228T165509, end_date=20231228T165512
[2023-12-28T16:55:12.941+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-28T16:55:13.122+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
